README.md

```diff
@@ -115,10 +115,11 @@ just the text of your transcript
 
 ### Getting captions output
 
-You can also get captions output from the SDK:
+You can also get captions output from the SDK. We offer both SRT and VTT caption formats.
+If you submitted your job as speaker channeled audio then you must also provide a `channel_id` to be captioned:
```

Comment:
```suggestion
If you submitted your job as speaker channel audio then you must also provide a `channel_id` to be captioned:
```

Reply:
Is this a word?

Reply:
I'm not sure but I think it gets the point across